server/src/test/java/io/crate/integrationtests/CursorITest.java

```diff
@@ -144,53 +144,63 @@ public void test_scroll_operations() throws Exception {
         try (var conn = DriverManager.getConnection(url(), properties)) {
             Statement statement = conn.createStatement();
             conn.setAutoCommit(false);
+            statement.execute("CREATE TABLE t1 AS SELECT * FROM generate_series(1,10,1)");
```

Comment:
To catch this case: https://github.com/crate/crate/pull/13481/files#diff-eb27fcfa3117c76e3f4a13fdd22325728cbdaf47b510c845508ed151dbbc2cceR213 had to create a concrete table.